phpunit.xml.dist

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Nitpick: do you mind aligning back all the attributes to where they were before your changes?